coalib/bearlib/languages/definitions/TinyBasic.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class TinyBasic:
+    aliases = 'tb',
+    extensions = '.tb',
+    comment_delimiters = 'REM',
```

Comment:
I just read that an apostrophe (') can be used in place of the REM command in tiny basic. If it's right, then you need to include it.

Reply:
I will go check that out, do you have a link that I could look at?

Reply:
I saw in this one http ://p112.sourceforge.net/tbp112.html

It also says that single quotes and double quotes both can be used for character strings. See to this one too.

Reply:
Nice find, seems legit to me. I will go ahead and read through it to double check for any other discrepancies, and I will push changes soon.